src/redux/store.js

```diff
@@ -0,0 +1,13 @@
+import { configureStore } from '@reduxjs/toolkit';
+import logger from 'redux-logger';
+import motorcycleReducer from './motorcylce/motorcylceSlice';
+import reservationReducer from './reservation/reservationSlice'
+
+export default configureStore({
+  reducer: {
+    motorcycle: motorcycleReducer,
+    reservation: reservationReducer,
+  },
+  // Middleware to show Redux Transactions: Please remove when we push to production
+  middleware: (getDefaultMiddleware) => getDefaultMiddleware().concat(logger),
```

Comment:
Middleware to show redux step-by-step transactions for debugging. Should be removed for production and only allowed to run if .env=DEVELOPMENT (future comment for deployment in the next few iterations)